test/box/net.box_incorrect_timeout_error_gh-6144.test.lua

```diff
@@ -0,0 +1,18 @@
+net = require('net.box')
```

Comment:
According to the [SoP](https://docs.google.com/document/d/1k1BqYup9SSbUwPQmNcgSp2M1LtRTQLc-), all new tests should be written in Luatest. Please consider migrating this test to Luatest. For an example, see [this test](https://github.com/tarantool/tarantool/blob/master/test/box-luatest/gh-6786_func_index_iterator_stable_test.lua).

Reply:
Fixed